cmd/snap/cmd_debug_state_test.go

```diff
@@ -325,3 +396,157 @@ func (s *SnapSuite) TestDebugIsSeededNo(c *C) {
 	c.Check(s.Stdout(), Matches, "false\n")
 	c.Check(s.Stderr(), Equals, "")
 }
+
+func (s *SnapSuite) TestDebugConnections(c *C) {
+	dir := c.MkDir()
+	stateFile := filepath.Join(dir, "test-state.json")
+	c.Assert(ioutil.WriteFile(stateFile, stateConnsJSON, 0644), IsNil)
+
+	rest, err := main.Parser(main.Client()).ParseArgs([]string{"debug", "state", "--connections", stateFile})
+	c.Assert(err, IsNil)
+	c.Assert(rest, DeepEquals, []string{})
+	c.Check(s.Stdout(), Matches,
+		"Interface       Plug                             Slot                            Notes\n"+
+			"desktop-legacy  gnome-calculator:desktop-legacy  core:desktop-legacy             auto\n"+
```

Comment:
I guess that this extra indentation is needed by gofmt :-(

Reply:
Yes, these tests changes were done by gofmt (excluding the additional tests, of course =p)